client/src/EventDetail.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { PollingEvent } from "./types.ts";
+
+type EventDetailProps = {
+    event: PollingEvent;
+};
+
+const EventDetail: React.FC<EventDetailProps> = ({ event }) => {
+    return (
+        <div>
+            <h2>{event.title}</h2>
+            <p>Místo: {event.location}</p>
+            <h3>Termíny</h3>
```

Comment:
Chybi ti tu zobrazeni aktualniho pocasi, pokud je misto (location). A pokud neni, asi by  to nemelo zobrazit `Misto: `